src/Symfony/Component/Cache/composer.json

```diff
@@ -18,7 +18,8 @@
     "provide": {
         "psr/cache-implementation": "1.0|2.0",
         "psr/simple-cache-implementation": "1.0",
-        "symfony/cache-implementation": "1.0|2.0"
+        "symfony/cache-implementation": "1.0|2.0",
+        "symfony/deprecation-contracts": "^2.1"
```

Comment:
this is wrong. `symfony/cache` does not provide `symfony/deprecation-contracts`

Reply:
right, fixed